README.md

```diff
@@ -45,7 +45,7 @@ Check out our [installation docs].
 ### Prerequisites
 
 **Note**: If you opt to not use `mise`, please install all dependencies
-from `.tool-versions` manually.
+from the `[tools]` section of `.mise.toml` manually.
```

Comment:
suggestion: use [GitHub-flavored Markdown admonitions](https://github.blog/changelog/2023-12-14-new-markdown-extension-alerts-provide-distinctive-styling-for-significant-content/)?
```suggestion
> [!NOTE]
> If you opt to not use `mise`, please install all dependencies
> from the `[tools]` section of `.mise.toml` manually.
```